src/ts/greedy-nav/GreedyNav.ts

```diff
@@ -0,0 +1,1062 @@
+import { blurEventName } from './helpers';
+import { Config, defaultConfig } from './Config';
+
+const root = window;
+
+const supports = !!document.querySelector && !!root.addEventListener; // Feature test
```

Comment:
Is there a reason `supports` is up here and not in the Class?

Reply:
The reason some methods are class methods and others not... I refactored functions that relied on Global variables to be instance methods so that the class could encapsulate the global state.

The remaining functions are 'outside' of the class. I don't think it benefits anyone to move them to `helpers.ts` - if anything I could move the contents of helpers inside the main file.

As mentioned this is part of a refactoring process which leaves the code working. It would be possible to improve this further but I'm making a judgement call of effort vs value.

Reply:
I removed the helpers.ts file - you're right - it wasn't providing any value, just added confusion. I think whomever (likely me, but who knows) continues to work on this can make a choice about a better structure for the code.